src/main/java/separator/Separator.java

```diff
@@ -0,0 +1,67 @@
+package separator;
+
+import java.util.ArrayList;
+
+public class Separator {
+    public static ArrayList<Character> separators = new ArrayList<>();
+
+    public Separator() {
+        separators.add(',');
+        separators.add(':');
+    }
+
+    public void saveSeparator(String input) {
```

Comment:
save는 뭔가 `저장하다` 느낌이지만 해당 메서드는 입력된 문자열에서 커스텀 구분자를 `추출하다`의 느낌 쪽에 비슷한 것 같아서 수정하면 알아보기 쉬울 것 같아요
```suggestion
    public void extractSeparator(String input) {
```

Reply:
다시 보니 기능을 완전히 설명하지 못한 영어 단어였던 것 같습니다ㅠㅠ 메소드 명을 지을 때 더 신중하게 지어보겠습니답:)